jolt-core/src/field/binius.rs

```diff
@@ -0,0 +1,185 @@
+use ark_serialize::{CanonicalDeserialize, CanonicalSerialize};
+use binius_field::{BinaryField128b, BinaryField128bPolyval};
+use std::ops::{Add, AddAssign, Div, Mul, MulAssign, Neg, Sub, SubAssign};
+
+use super::JoltField;
+
+impl BiniusConstructable for BinaryField128b {
+    fn new(n: u64) -> Self {
+        Self::new(n as u128)
+    }
+}
+
+impl BiniusConstructable for BinaryField128bPolyval {
+    fn new(n: u64) -> Self {
+        Self::new(n as u128)
+    }
+}
+
+impl BiniusSpecific for BinaryField128b {}
+impl BiniusSpecific for BinaryField128bPolyval {}
+
+/// Trait for BiniusField functionality specific to each impl.
+pub trait BiniusSpecific: binius_field::TowerField + BiniusConstructable + bytemuck::Pod {}
+
+pub trait BiniusConstructable {
+    fn new(n: u64) -> Self;
+}
+
+#[derive(Default, Debug, Copy, Clone, Eq, PartialEq)]
+pub struct BiniusField<F: BiniusSpecific>(F);
+
+/// Wrapper for all generic BiniusField functionality.
+impl<F: BiniusSpecific> JoltField for BiniusField<F> {
+    const NUM_BYTES: usize = 16;
+
+    fn random<R: rand_core::RngCore>(rng: &mut R) -> Self {
+        Self(F::random(rng))
+    }
+
+    fn is_zero(&self) -> bool {
+        self.0.is_zero().into()
+    }
+
+    fn is_one(&self) -> bool {
+        self.0 == Self::one().0
+    }
+
+    fn zero() -> Self {
+        Self(F::ZERO)
+    }
+
+    fn one() -> Self {
+        Self(F::ONE)
+    }
+
+    fn from_u64(n: u64) -> Option<Self> {
+        Some(Self(F::new(n)))
+    }
+
+    fn square(&self) -> Self {
+        Self(self.0.square())
+    }
+
+    fn from_bytes(bytes: &[u8]) -> Self {
+        assert_eq!(bytes.len(), Self::NUM_BYTES);
+
+        let field_element = bytemuck::try_from_bytes::<F>(bytes).unwrap();
+        Self(field_element.to_owned())
+    }
+}
+
+impl<F: BiniusSpecific> Neg for BiniusField<F> {
+    type Output = Self;
+
+    fn neg(self) -> Self {
+        BiniusField(self.0.neg())
+    }
+}
+
+impl<F: BiniusSpecific> Add for BiniusField<F> {
+    type Output = Self;
+
+    fn add(self, other: Self) -> Self {
+        BiniusField(self.0.add(other.0))
```

Comment:
same thing here and below
```suggestion
        BiniusField(self.0 + other.0)
```